iree/compiler/InputConversion/MHLO/MHLOToLinalgOnTensors.cpp

```diff
@@ -102,14 +101,11 @@ struct ConcatenateOpConversion
     return success();
   }
 };
-}  // namespace
 
 //===----------------------------------------------------------------------===//
 // mhlo.fft conversion patterns.
 //===----------------------------------------------------------------------===//
 
-namespace {
```

Comment:
Why are we losing internal linkage?

Reply:
It just looks that way: this anonymous namespace was nested in an anonymous namespace.